### ISSUE TITLE
Connection error

### ISSUE THREAD

**Issue by Phoenix-DH** [materik/homebridge-harmonyhub-plugin#10]
Hi,

I tried your version, but I get also this error message:
```
errorhub { Error: connect ECONNREFUSED 192.168.0.10:5222
at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1117:14)
errno: 'ECONNREFUSED',
code: 'ECONNREFUSED',
syscall: 'connect',
address: '192.168.0.10',
port: 5222 }
```

Buttons in homebkit not working.

**Comment by stefantitu** [comment]
Same here

**Comment by materik** [comment]
@Phoenix-DH @stefantitu I made a fix for #8 that might solve this issue as well. Could you try installing ```npm install -g materik/homebridge-harmonyhub#develop``` and see if that works for you?

**Comment by materik** [comment]
Closing this since I haven't heard of the problem persisting.
